### PR TITLE
Unique vendorId for 40percentclub boards and via keymaps for 4x4 and 5x5

### PR DESCRIPTION
**Pull request by devilzmods**
## Description

I want to add both boards to Via and need the proper changes integrated into qmk. While I was at it, I took the liberty of changing the default vendorId 0xFEED on all 40percentclub boards to 0x3430 which is already used in the sixpack, nein and mf68's Via layouts

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/40percentclub/25/config.h [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x3430
```
Comment (drashna) [inline keyboards/40percentclub/25/config.h:22]: This vendor ID is already in use and needs to be changed. 

https://www.the-sz.com/products/usbid/index.php?v=0x3430&p=&n=

Ideally, anything above 0x4000 is fairly safe, but the higher, the better.

File: keyboards/40percentclub/4x4/keymaps/via/4x4.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (drashna) [inline keyboards/40percentclub/4x4/keymaps/via/4x4.json:1]: This is a VIA json and as such needs to be removed, and pr'ed to `the-via/keyboards` instead.

File: keyboards/40percentclub/4x4/keymaps/via/config.h [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (drashna) [inline keyboards/40percentclub/4x4/keymaps/via/config.h:1]: This file should be removed.

File: keyboards/40percentclub/4x4/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,104 @@
+
```
Comment (drashna) [inline keyboards/40percentclub/4x4/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/40percentclub/4x4/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+#include QMK_KEYBOARD_H
+
+#define PAD 0
+#define _QW 1
+#define NUM 2
+#define DIR 3
```
Comment (drashna) [inline keyboards/40percentclub/4x4/keymaps/via/keymap.c:7]:
```suggestion
enum custom_layers {
    PAD,
    _QW,
    NUM,
    DIR,
};
```

File: keyboards/40percentclub/4x4/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  return true;
+}
+
+void led_set_user(uint8_t usb_led) {
+
+}
```
Comment (drashna) [inline keyboards/40percentclub/4x4/keymaps/via/keymap.c:104]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]
And same for all the changed files that I didn't flag (such as the 5x5via keymap)

File: keyboards/40percentclub/25/config.h (hunk 1 above)
Comment (devilzmods) [reply keyboards/40percentclub/25/config.h]: Picked 0x4025, which can read as 40 %

File: keyboards/40percentclub/4x4/keymaps/via/keymap.c (hunk 4 above)
Comment (devilzmods) [reply keyboards/40percentclub/4x4/keymaps/via/keymap.c:16]: Done, smae with the 5x5

**Review by drashna — APPROVED** [review]

File: keyboards/40percentclub/4x4/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+ * |        | F1     | F2     | F3     | F4     | F5     | F6     | F7     | F8     | F9     | F10    | NUMLOCK|   /    |   *    |   -    |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * |    `   | 1      | 2      | 3      | 4      | 5      | 6      | 7      | 8      | 9      | 0      |        |        |        |   +    |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * |        | F11    | F12    |        |        |        | ENTER  | SHIFT  | RGUI   | ./ALT  | BKSC   |        |        |        | ENTER  |        |
+ * |        |        |        |        |        |        |        |        |        |        |CTRLhold|		   |        |        |        |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|        |
```
Comment (fauxpark) [inline keyboards/40percentclub/4x4/keymaps/via/keymap.c:74]:
```suggestion
 * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
```

File: keyboards/40percentclub/tomato/config.h [hunk 8]
```diff
@@ -3,8 +3,8 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6060
+#define VENDOR_ID       0x4025
+#define PRODUCT_ID      0x546f //"To", was duplicate with polyandry
```
Comment (fauxpark) [inline keyboards/40percentclub/tomato/config.h:7]:
```suggestion
#define PRODUCT_ID      0x546F //"To", was duplicate with polyandry
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**